templates/taxi/car_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🔍">
+  </form>
```

Comment:
Create separate template for search form and use it where you need

Reply:
How is that? I did not quite understand

Reply:
You have separate template for pagination and sidebar. You need to create the same template for search form and use it in other templates